lib/challenge.rb

```diff
@@ -16,21 +21,12 @@ def each
       next_fib = current_fib + previous_fib
     end
   end
-
-  attr_reader :stop
-  private :stop
```

Comment:
If someone naively uses `stop`, and it's private, it errors. That tells the person "don't do that".

Someone can introspect an instance, see a stop method if its public, and start playing with it. Not a problem here, but you can see why not considering whether something should be private or not could be a problem.

That's why I choose personally to default to making it private until I know how I want the API exposed.